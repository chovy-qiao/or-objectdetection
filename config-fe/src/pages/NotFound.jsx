import React from 'react';
import logo from 'assets/notfound.gif'

class NotFound extends React.Component {
    render() {
        return(
            <div>
                <img src={logo} alt="loading..." />
            </div>
        )
    }
}

export default NotFound;