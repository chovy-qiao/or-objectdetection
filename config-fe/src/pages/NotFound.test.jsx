import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import NotFound from "./NotFound";
import { MemoryRouter } from "react-router-dom";

const renderNotFound = () => {
  return render(
    <MemoryRouter>
      <NotFound />
    </MemoryRouter>
  );
};
describe("Configuration component", () => {
  it("matches the snapshot", () => {
    const { container } = renderNotFound();
    expect(container).toMatchSnapshot();
  });
});
