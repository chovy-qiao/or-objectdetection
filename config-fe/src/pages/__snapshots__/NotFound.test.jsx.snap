// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Configuration component matches the snapshot 1`] = `
<div>
  <div>
    <img
      alt="loading..."
      src="notfound.gif"
    />
  </div>
</div>
`;
